dashboard/config/routes.rb

```diff
@@ -62,7 +62,10 @@
 
   get 'redirected_url', to: 'redirect_proxy#get', format: false
 
-  get 'docs/', to: 'curriculum_proxy#get_doc_landing'
+  get 'docs/', to: 'programming_environments#docs_index'
```

Comment:
Are there unit tests that could be added to check that all of these URLs go to the right place and that ones that should go to CB for concept maps aren't impacted?

Reply:
I forgot to add one for this route but the fact that [this test](https://github.com/code-dot-org/code-dot-org/blob/fd12d7bac947944cf3f138722df151b8aeade634/dashboard/test/controllers/curriculum_proxy_controller_test.rb#L17) still passes is evidence that concept maps aren't affected.

Reply:
Great. Thanks for pointing to existing tests that are covering this